docs/config/README.md

```diff
@@ -100,6 +100,7 @@ can do this in two ways: by changing color (red/green) or by changing its shape
 | `symbol`                | `"➜"`   | The symbol used before the text input in the prompt. |
 | `error_symbol`          | `"✖"`   | The symbol used before text input if the previous command failed. |
 | `use_symbol_for_status` | `false` | Indicate error status by changing the symbol.         |
+| `vicmd_symbol`          | `"❮"`   | The symbol used before the text input in the prompt if Zsh Line Editor keymap is `vicmd`. |
```

Comment:
As discussed on #130, this might not work well with our colorblind support. It might be better to add a new module to indicate vi-status.

@matchai I guess we have to decide now. Do we want to overwrite the prompt symbol or use a new module? FWIW, on testing this, it looks like after running a command, you will initially have the success/failure symbol, and it only changes to the vi symbol if you actually go into normal mode.

Reply:
I'm in favor of replacing the symbol. Perhaps we could have a colorblind mode that will prefix the character symbol with a "✖" on a non-zero status, and leave a space for it otherwise: ` ` `➜` or `✖➜`.

Reply:
I think I'd be happy with leaving this as it is for now then. You'll lose return status info if you switch into normal mode, but I don't think that information is so crucial that it has to be preserved in the scrollback history.

Reply:
If we need to, we can add a colorblind mode later, though I'd prefer to review the entire shell to make sure that everything is colorblind-friendly before committing to it as an option.